Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -192,6 +199,177 @@ def download_export_scan(self, scan_id: str, file_id: str, file_format: str) ->
+            from_date: date to fetch audit logs from.
+            to_date: date which until to fetch audit logs.
+            actor_id: fetch audit logs with matching actor id.
+            target_id:fetch audit logs with matching target id.
+
+        Returns:
+            audit logs fetched from the API.
+        """
+        query = ''
+        if from_date:
+            query = self.add_query(query, f'f=date.gt:{from_date}')
+        if to_date:
+            query = self.add_query(query, f'f=date.lt:{to_date}')
+        if actor_id:
+            query = self.add_query(query, f'f=actor_id.match:{actor_id}')
+        if target_id:
+            query = self.add_query(query, f'f=target_id.match:{target_id}')
+        if limit:
+            query = self.add_query(query, f'limit={limit}')
+        else:
+            query = self.add_query(query, 'limit=5000')
+        res = self._http_request(method='GET', url_suffix=f'/audit-log/v1/events{query}', headers=self._headers)
+        return res.get('events', [])
+
+    def get_vuln_export_uuid(self, num_assets: int, last_found: Optional[float], severity: List[str]):
+        """
+
+        Args:
+            num_assets: number of assets used to chunk the vulnerabilities.
+            last_found: vulnerabilities that were last found between the specified date (in Unix time) and now.
+            severity: severity of the vulnerabilities to include in the export.
+
+        Returns: The UUID of the vulnerabilities export job.
+
+        """
+        payload: dict[str, Any] = {
+            "filters":
+                {
+                    "severity": severity,
+                    "last_found": last_found
+                },
+            "num_assets": num_assets
+        }
+
+        res = self._http_request(method='POST', url_suffix='/vulns/export', headers=self._headers, json_data=payload)
+        return res.get('export_uuid', '')
+
+    def get_vuln_export_status(self, export_uuid: str):
+        """
+
+        Args:
+            export_uuid: The UUID of the vulnerabilities export job.
+
+        Returns: The status of the job, and number of chunks available if succeeded.
+
+        """
+        res = self._http_request(method='GET', url_suffix=f'/vulns/export/{export_uuid}/status',
+                                 headers=self._headers)
+        status = res.get('status')
+        chunks_available = res.get('chunks_available', [])
+        return status, chunks_available
+
+    def download_vulnerabilities_chunk(self, export_uuid: str, chunk_id: int):
+        """
+
+        Args:
+            export_uuid: The UUID of the vulnerabilities export job.
+            chunk_id: The ID of the chunk you want to export.
+
+        Returns: Chunk of vulnerabilities from API.
+
+        """
+        return self._http_request(method='GET', url_suffix=f'/vulns/export/{export_uuid}/chunks/{chunk_id}',
+                                  headers=self._headers)
+
+    def export_assets_request(self, fetch_from):
```

Comment:
I would change the name to get_asset_export_uuid
or 
request_asset_export_uuid

When the export is the first word it sounds like this method is exporting something